packages/charts/chart-utilities/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "dist",
+    "target": "es5",
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "strictNullChecks": true,
+    "strictBindCallApply": true,
+    "noImplicitAny": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
+    "skipLibCheck": true,
+    "lib": ["es5", "dom"],
+    "typeRoots": ["../../../node_modules/@types", "../../../typings"],
+    "types": ["jest", "custom-global", "@testing-library/jest-dom"],
```

Comment:
fyi: this setup is deprecated and shouldn't be used as it creates various of issues starting with leaking cross domain scopes to production packages.